website/Contribution/oca_repository_policy.rst

```diff
@@ -0,0 +1,420 @@
+##################################
+|TITLE| (|DATE|)
+##################################
+
+.. |TITLE| replace:: OCA Repository Policy
+.. |DATE| replace:: 2018/08/24
+.. |RELATED| replace:: OCA Module Lifecycle - Maintainer Role
+.. |CATEGORIES| replace:: Contribution, Repository Management, Policy
+
+.. fill in the Document information in the above lines. It will be automatically
+   replaced in the document header. Don't forget to remove the []
+
+.. contents:: Table of contents
+    :depth: 4
+
+.. sectnum::
+
```

Comment:
@elicoidal Can you insert a word saying the canonical URL to this document is https://odoo-community.org/page/development-status

I'll adapt the redirection when it's merged.

Reply:
@elicoidal sorry, my above comment does not make sense, I though it was the development status document (maturity level), but now I see there is no PR for that one. https://odoo-community.org/page/development-status

Reply:
I will remove the footer and reference to this link